datadog_checks_dev/datadog_checks/dev/tooling/commands/release/trello/testable.py

```diff
@@ -113,11 +113,7 @@ def _all_synced_with_remote(refs: Sequence[str]) -> bool:
 def _get_and_parse_commits(base_ref: str, target_ref: str) -> List[Tuple[str, str]]:
     echo_info(f'Getting diff between {base_ref!r} and {target_ref!r}... ', nl=False)
 
-    # Format as '<commit_hash> <subject line>', e.g.:
-    # 'a70a792f9d1775b7d6d910044522f7a0d6941ad7 Update README.md'
-    pretty_format = '%H %s'
-
-    diff_command = f'git --no-pager log "--pretty={pretty_format}" {base_ref}..{target_ref}'
+    diff_command = f'git --no-pager cherry -v {base_ref} {target_ref}'
```

Comment:
(nit) Maybe we can update the Format comment with `git cherry -v` output ?

I can it can help people reading this section of the code.